src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointNavigation.js

```diff
@@ -30,22 +30,24 @@ function BreakpointNavigation({
       seek(point.point, point.time, true);
     }
   };
-  const isEmpty = analysisPoints && (analysisPoints.error || analysisPoints.data.length == 0);
+  const points = analysisPoints?.data || [];
```

Comment:
It's unfortunately how many untyped files we have in our codebase (combined with kind of minimal e2e test coverage).